### ISSUE TITLE
Import IATI XML

### ISSUE THREAD

**Issue by stevieflow** [younginnovations/aidstream-old#44]
Suggestion:

For users who have generated their (valid) IATI XML elsewhere, a function to import this into AidStream and populate the relevant fields - would be very useful.

**Comment by siemvaessen** [comment]
Yes. It would be of great added value if Aidstream could validate this inline. Perhaps even mapping a non valid (wanttobe) IATI file (in csv for example) and have Aidstream provide a mapping service would be nice to have.

**Comment by stevieflow** [comment]
Yes, agree.

In this use case, Im thinking of organisations that _already_ have IATI XML and wish to start up / migrate to AidStream publishing.  The requirements would be that this XML would be valid against the schema, and perhaps also contain valid code list values (as AidStream would not store non-valid values).  The version attribute may also cause issues
